modules/migration-mtc-release-notes-1-8-4.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-4_{context}"]
+= {mtc-full} 1.8.4 release notes
+
+[id=technical-changes-1-8-4_{context}]
+== Technical changes
+
+{mtc-full} ({mtc-short}) 1.8.4 has the following technical changes:
+
+* {mtc-short} 1.8.4 extends its dependency resolution to include support for using {oadp-first} 1.4.
+
+.Support for KubeVirt Virtual Machines with DirectVolumeMigration
+
+{mtc-short} 1.8.4 adds support for KubeVirt Virtual Machines (VMs) with Direct Volume Migration (DVM).
+
+[id="resolved-issues-1-8-4_{context}"]
+== Resolved issues
+
+{mtc-full} ({mtc-short}) 1.8.4 has the following major resolved issues:
+
+.Ansible Operator is broken when {VirtProductName} is installed
+
+There is a bug in the `python3-openshift` package that installing {VirtProductName} exposes, with an exception, `ValueError: too many values to unpack`, returned during the task. Earlier version of {mtc-short} are impacted, while {mtc-short} 1.8.4 has implemented a workaround. Updating to {mtc-short} 1.8.4 means you are no longer affected by this issue. link:https://issues.redhat.com/browse/OCPBUGS-38116[(OCPBUGS-38116)]
+
+.UI stuck at Namespaces while creating a migration plan
+
+When trying to create a migration plan from the {mtc-short} UI, the migration plan wizard becomes stuck at the *Namespaces* step. This issue has been resolved in {mtc-short} 1.8.4. link:https://issues.redhat.com/browse/MIG-1597[(MIG-1597)]
+
+.Migration fails with error of no matches for kind Virtual machine in version kubevirt/v1
+
+During the migration of an application, all the necessary steps, including the backup, DVM, and restore, are successfully completed. However, the migration is marked as unsuccessful with the error message `no matches for kind Virtual machine in version kubevirt/v1`. link:https://issues.redhat.com/browse/MIG-1594[(MIG-1594)]
+
+.Direct Volume Migration fails when migrating to a namespace different from the source namespace
+
+On performing a migration from source cluster to target cluster, with target namespace different from the source namespace, the DVM fails. link:https://issues.redhat.com/browse/MIG-1592[(MIG-1592)]
```

Comment:
I feel like there should be an article before target namespace, such as "the" or "a":
```suggestion
On performing a migration from source cluster to target cluster, with the target namespace different from the source namespace, the DVM fails. link:https://issues.redhat.com/browse/MIG-1592[(MIG-1592)]
```